### PR TITLE
CEDS-1162: Supervising customs office & office of presentation (5/26 & 5/27)

### PR DESCRIPTION
**Pull request by benjaminparker**

### CODE REVIEW

File: app/services/model/CustomsOffice.scala [hunk 1]
```diff
@@ -14,14 +14,6 @@
...
-trait SupervisingCustomsOfficeMessages {
-
-  val supervisingCustomOffice: String = "supplementary.supervisingCustomsOffice"
-
-  val title: String = supervisingCustomOffice + ".title"
-  val hint: String = supervisingCustomOffice + ".hint"
-  val scoError: String = supervisingCustomOffice + ".error"
-  val scoEmpty: String = supervisingCustomOffice + ".empty"
-}
+case class CustomsOffice(code: String, description: String)
```
Comment (ed0906) [inline app/services/model/CustomsOffice.scala]: Is this meant to be in test code?

Comment (ed0906) [reply app/services/model/CustomsOffice.scala]: Ignore me, didnt noticed the package move too

File: app/services/view/AutoCompleteItem.scala [hunk 2]
```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
-package services.model
+package services.view
```
Comment (ed0906) [inline app/services/view/AutoCompleteItem.scala]: If we're moving this, i'd suggest it goes into the `views.components.inputs` package as its only relevant to the `input_radio` and stuff that uses that

File: test/controllers/declaration/WarehouseIdentificationControllerSpec.scala [hunk 3]
```diff
@@ -46,34 +46,12 @@ class WarehouseIdentificationControllerSpec extends CustomExportsBaseSpec with W
       status(result) must be(OK)
     }
 
-    "display correct hints and questions" in {
```
Comment (ed0906) [inline test/controllers/declaration/WarehouseIdentificationControllerSpec.scala]: Have we agreed not to have these style tests?

Comment (PatrykRudnicki) [reply test/controllers/declaration/WarehouseIdentificationControllerSpec.scala]: It's inside controller. This is covered in view spec

File: app/services/view/AutoCompleteItem.scala (hunk 2 above)
Comment (benjaminparker) [reply app/services/view/AutoCompleteItem.scala]: services.views => views is usually only for .html.scala files
services.model is needed for the backend CSV parsing

Was discussed with Matt / Pat and agreed services.view ....

File: test/controllers/declaration/WarehouseIdentificationControllerSpec.scala (hunk 3 above)
Comment (benjaminparker) [reply test/controllers/declaration/WarehouseIdentificationControllerSpec.scala]: Perhaps - I wasn't in the test meeting.  Can move to the view spec if required or remove?

Comment (PauloPortugal) [reply test/controllers/declaration/WarehouseIdentificationControllerSpec.scala]: These should be in the `WarehouseIdentificationViewSpec.scala`, which I think we already have, so this code is redundant.
Looks good.

**Comment by PauloPortugal** [comment]
Please resolve conflicts

File: test/controllers/declaration/WarehouseIdentificationControllerSpec.scala (hunk 3 above)
Comment (benjaminparker) [reply test/controllers/declaration/WarehouseIdentificationControllerSpec.scala]: It was testing code that was already in a ViewSpec

File: app/services/view/AutoCompleteItem.scala (hunk 2 above)
Comment (ed0906) [reply app/services/view/AutoCompleteItem.scala]: "views is usually only for .html.scala files" Just my view but anything view related should go in the `views` package, that includes AutoComplete, RadioOption and whatever other view helpers we have

Comment (ed0906) [reply app/services/view/AutoCompleteItem.scala]: At the moment we're using the `services` package rather like a `utils` package

**Pull request merged**